app/routes/editor/new.js

```diff
@@ -36,6 +37,12 @@ export default AuthenticatedRoute.extend(base, {
         psm.send('resetPubDate');
 
         this._super(...arguments);
+
+        controller.set('cards' , []);
+        controller.set('atoms' , []);
+        controller.set('toolbar' , []);
+        controller.set('apiRoot', ghostPaths().apiRoot);
```

Comment:
Yep,

As Kevin said above the whole editor is a bit of a mess and it's not 100% clear that everything is behaving as it should. I'll also adjust the edit route.